atom/catom.pyi

```diff
@@ -75,14 +75,16 @@ class Member(Generic[T, S]):
     def clone(self: M) -> M: ...
     def add_static_observer(
         self,
-        observer: str | Callable[..., None],
+        observer: str | Callable[[ChangeDict], None],
```

Comment:
Actually a static observer takes a reference to the owning atom object, see atom.py for an example (tldr you need a TypeVar to work around the function contravariance)

Reply:
Also the fact that the test pass tell me I should improve the type validation tests.

Reply:
PyObject_Call must do some magic with the "self" pointer when using a method vs non-method because the test uses
```python
  def react(change):
        changes.append(change)
```

But changing it to this throws an error
```python
  def react(obj, change):
        changes.append(change)
```

Reply:
My bad ! observe can be used on such a callable when it is a method, but at call time it retrieves it by name to get only the bound version.

Sorry for the noise